src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -21,16 +21,18 @@
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.team2412.robot.Robot;
+import frc.team2412.robot.subsystem.ClimbSubsystem.ClimbConstants.AutoClimbState;
 import frc.team2412.robot.subsystem.ClimbSubsystem.ClimbConstants.SolenoidState;
 import io.github.oblarg.oblog.Loggable;
+import io.github.oblarg.oblog.annotations.Config;
 import io.github.oblarg.oblog.annotations.Log;
 
 public class ClimbSubsystem extends SubsystemBase implements Loggable {
 
     public static class ClimbConstants {
         public static final double MAX_SPEED = 1;
```

Comment:
It would be good to reduce max to much less than one to make sure climb does not break.  If not used, it should be removed.